pkg/controller/node/nodecontroller.go

```diff
@@ -489,6 +490,9 @@ func (nc *NodeController) monitorNodeStatus() error {
 			// Report node event.
 			if readyCondition.Status != api.ConditionTrue && lastReadyCondition.Status == api.ConditionTrue {
 				nc.recordNodeStatusChange(node, "NodeNotReady")
+				if err = nc.markAllPodsNotReady(node.Name); err != nil {
```

Comment:
Looks like it runs every 5 seconds... The timeout for node notready is 40 secs. So the worst case is 45 secs to flip node to notready.